.github/workflows/ci.yml

```diff
@@ -22,5 +22,9 @@ jobs:
       - name: Fetch npm dependencies
```

Comment:
What is the overall data flow of a user using full circle?

Reply:
Maybe the readme clears this up now?